<launch>
	<!-- Send to drone -->
	<node pkg ="Communication" type="wifi_node" name="topic_in_udp_out" output="screen">
		<param name="ip" value ="192.168.0.15" />
		<param name="port" value="8888" />
		<param name="topic_sub" value="/communication/to_drone4" />
	</node>

	<!-- Capture from cam/file -->
        <node pkg="markerLocator" name = "cam_to_topic" type="cam_to_topic.py" output="screen"  >
                <!--param name="device" value = "/home/exchizz/SDU/Skole/Bachelor/PC/ROS/src/Bachelor_ROS/Tracking/markerLocator/src/my_video-10.mkv" /-->
		<param name="device" value="0" />
		<param name="camera_out" value="/camera/down" />
        </node>

	<!-- Track order 4 -->
        <node pkg="markerLocator" name = "MarkerLocator_track4" type="MarkerLocator_node.py" output="screen"  >
                <param name="camera_in" value="/camera/down" />
                <param name="to_find" value="6" />
		<param name="pose_out" value="/positions/drone4" />
	</node>

	<node pkg="decisionMaker" name="Decision_Marker" type="decision_maker_node.py" output="screen" >
		<param name="drone_out" value="/communication/to_drone4" />
		<param name="pose_in" value="/positions/drone4" />
	</node>
</launch>
